cli/main.rs

```diff
@@ -532,9 +533,36 @@ async fn run_repl(flags: Flags) -> Result<(), ErrBox> {
 
 async fn run_command(flags: Flags, script: String) -> Result<(), ErrBox> {
   let global_state = GlobalState::new(flags.clone())?;
-  let main_module = ModuleSpecifier::resolve_url_or_path(&script).unwrap();
+  let main_module = if script != "-" {
+    ModuleSpecifier::resolve_url_or_path(&script).unwrap()
+  } else {
+    ModuleSpecifier::resolve_url_or_path("./__$deno$run.ts").unwrap()
```

Comment:
Maybe stdin instead of run?

Reply:
All the others use` __$deno${command}.ts` for fake files used by deno commands, so I decided to follow, but it is a special case within run. I'd say its more consistent to use `__$deno$run.ts` but it doesn't really matter.

Reply:
changed it to stdin.ts